mqtt_as/README.md

```diff
@@ -691,6 +695,125 @@ async def messages(client):
 In applications RAM is at a premium, in testing the callback-based interface
 offers somewhat (~1.3KiB) lower consumption than the minimal queue case.
 
+## 3.6 MQTTv5 Support
+### 3.6.1 Configuration and Migration from MQTTv3.1.1
+MQTTv5 is supported and can be configured by setting `mqttv5` to `True` in the
+the `config` dictionary. The default is `False`. Properties on connect
+are supported, these need to be passed in the configuration dictionary. See
+3.6.2 for more information on properties, and how to format them.
+
+
+```python
+from mqtt_as import MQTTClient, config
+config['mqttv5'] = True
+
+# Optional: Set the properties for the connection
+config['mqttv5_con_props'] = {
+    0x11: 3600,  # Session Expiry Interval
+}
+
+# The rest of the configuration
+...
+
+client = MQTTClient(config)
+
+```
+
+There are modifications to the API to support MQTTv5 features. The most
+significant is the addition of the `properties` argument that is provided as
+an additional argument to both the event and callback-based message handlers.
+
+```python
+# For MQTT 3.1.1 support
+def callback(topic, msg, retained):
+    print((topic, msg, retained))
+
+# For MQTT 5 and 3.1.1 support
+def callback(topic, msg, retained, properties=None):
+    print((topic, msg, retained, properties))
+```
+
+Allowing properties as an optional argument allows you to switch between
+MQTT 3.1.1 and MQTT 5 support without changing the callback signature.
+
+```python
+async def messages(client):
+    async for topic, msg, retained, properties in client.queue:
+        await asyncio.sleep(0)  # Allow other instances to be scheduled
+        # handle message
+```
+
+The `properties` argument is a dictionary that contains the properties of the
+message. The properties are defined in the MQTTv5 specification. If you 
+include properties in published messages, while using MQTTv3.1.1, the properties
+will be ignored.
+
+### 3.6.2 MQTTv5 Properties
+Properties are a new and important feature of MQTTv5. They are used to provide
+additional information about the message, and allow for more advanced features
+such as message expiry, user properties, and response information.
+
+Incoming properties are formatted as a dictionary using the property  identifier
+as the key. The property identifier is an integer that is defined in the MQTTv5
```

Comment:
There seems to be a missing word here. Should this read "in the MQTTv5 specification"?

Reply:
Yes, you are correct. It is fixed now.